prow/github/types.go

```diff
@@ -1260,6 +1260,7 @@ type GenericCommentEvent struct {
 type Milestone struct {
 	Title  string `json:"title"`
 	Number int    `json:"number"`
+	State  string `json:"state"`
```

Comment:
GitHub seems to include this field by default on milestones and we already serialize the whole struct. But, can you elaborate more on where the state will be utilized?

Reply:
There is the [ListMilestones](https://github.com/kubernetes/test-infra/blob/9c351a065a9701b0d3a683308ddc6f7b2d1edcf0/prow/github/client.go#L4598) method, but it currently does not return whether the `Milestone` is open or not.